README.md

```diff
@@ -1,40 +1,40 @@
 # Shell L1 Driver Standard
 
-**Usage:**
+**Creation and configuration of the driver's environment:**
 
 1. Start a new project with [shellfoundry](https://github.com/QualiSystems/shellfoundry):
 ```bash
 $ shellfoundry new DriverName --template layer-1-switch
 ```
-2. Install requirements (from the new project folder):
+2. Create a new Python virtualenv in the project's folder. Specify the same python executable which is used by CloudShell.
+```bash
+virtualenv --python="c:\Program Files (x86)\QualiSystems\CloudShell\Server\python\2.7.10\python.exe" --always-copy .\cloudshell-L1-DriverName
+```
+3. Activate the project's virtualenv
+```bash
+.\cloudshell-L1-DriverName\Scripts\activate.bat
+```
+4. Edit and install requirements (from the new project folder):
 ```bash
 $ pip install -r requirements.txt
 
 ```
+**Implementation:**
 
-3. Implement methods of DriverCommands class in <project_slug>/driver_commands.py. Follow [DEVGUIDE](https://github.com/QualiSystems/shell-L1-standard/blob/dev/DEVGUIDE.md) and docstrings with description, as an example of L1 driver with CLI usage you can use [cloudshel-L1-mrv](https://github.com/QualiSystems/cloudshell-L1-mrv)
-
-4. Update the driver version and metadata in version.txt
-
-**Compile the driver:**
+1. Implement methods of the DriverCommands class in <project_slug>/driver_commands.py. Follow the [DEVGUIDE](https://github.com/QualiSystems/shell-L1-standard/blob/dev/DEVGUIDE.md) and docstrings with description, as an example of an L1 driver with CLI usage you can reffer to the [cloudshel-L1-mrv](https://github.com/QualiSystems/cloudshell-L1-mrv) project.
+To debug the driver use the [DEBUGGING GUIDE](https://github.com/QualiSystems/shell-L1-template/blob/run_driver_without_pyinstaller/DEBUGGING.md).
 
-1. download required packages from requirements.txt and uncompress them at the same level as the project folder (next to it). Note: [cloudshell-core](https://github.com/QualiSystems/cloudshell-core) and [cloudshell_l1_networking_core](https://github.com/QualiSystems/cloudshell-L1-networking-core) are required packages and must have the next paths:
+2. Update the driver version and metadata in version.txt
 
-cloudshell-core package: "../cloudshell-core"
-
-cloudshell_l1_networking_core package: "../cloudshell-L1-networking-core"
-
-2. [install pyinstaller](http://pyinstaller.readthedocs.io/en/latest/installation.html):
-```bash
-$ pip install pyinstaller
-
-```
 
-3. run compile_driver.bat (the compiled driver will be created in a "dist" directory)
+**Compile the driver executable and build package:**
+1. Download [Bat To Exe](http://www.f2ko.de/en/b2e.php) Converter
+2. Compile *DriverName.bat* to *DriverName.exe*
+4. Create new zip archive with the *Cloudshell-L1-DriverName* project folder and *Cloudshell-L1-DriverName\DriverName.exe* executable inside, located both at the same level.
```

Comment:
where is 3.? about how to use b2e (it can also be under 2. but then change 4. to 3.)